nuxeo-services/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/NuxeoAuthenticationFilter.java

```diff
@@ -299,14 +299,25 @@ protected Principal doAuthenticate(CachableUserIdentificationInfo cachableUserId
 
             logAuthenticationAttempt(cachableUserIdent.getUserInfo(), true);
         } catch (LoginException e) {
-            log.info("Login failed for " + cachableUserIdent.getUserInfo().getUserName());
+            if (log.isInfoEnabled()) {
+                log.info(String.format("Login failed for %s on request %s",
+                        cachableUserIdent.getUserInfo().getUserName(), httpRequest.getRequestURI()));
+            }
+            log.debug(e, e);
```

Comment:
do we want this log.debug ?
also is should we check for NPE on cachableUserIndent.getUserInfo() ?

Reply:
Yes, adding the debug log is the main purpose of this ticket, otherwise, we can never know the cause of the authentication failure.

We can add an NPE check yes, but this was just some refactoring.